### ISSUE TITLE
Messenger Notifications not working

### ISSUE THREAD

**Issue by CrashyBang** [saenzramiro/rambox#464]
### Steps to reproduce

1. Open Rambox
2. Receive message on Messenger
3. No notification received 

### Expected behaviour

Expect to get a notification.

### Actual behaviour

No notification is received, however Messenger notifications are coming through in chrome so to my understanding they have been enabled correctly?

### ENVIRONMENT

> Rambox: v0.4.4
> OS: Ubuntu 16.04

> Arch: x64

**Comment by saenzramiro** [comment]
When you add Facebook Messenger service, there is a note in the window that said you have to enable notifications in Facebook Messenger settings.

**Comment by CrashyBang** [comment]
As per actual behaviour I have done that, we I tried to do it in Rambox itself nothing changed, so I enabled through chrome and it worked as expected.

**Comment by saenzramiro** [comment]
Please check if Don't Disturb mode is OFF and "Show notifications" option in Messenger service is algo enabled.

**Comment by CrashyBang** [comment]
Same issue with the above settings enabled.

**Comment by saenzramiro** [comment]
And only with Messenger?

Try removing service and add it again, then enable Notifications inside Messenger Settings.

**Comment by CrashyBang** [comment]
Hey @saenzramiro okay will do, one question should I be enabling notifications from inside rambox or inside the browser, when I did try to enable them in rambox there was no popup box enabling me to allow them?

**Comment by saenzramiro** [comment]
No, in Rambox there's no popup to allow them like Chrome.

Follow this steps:

1. Add Facebook Messenger service.
2. Log in.
3. Inside Facebook Messenger, go to the cog at the top left corner and select Settings.
4. Then click in "Activate notifications" option.

**Comment by CrashyBang** [comment]
Sorted :). Might fire through a PR that makes the message a little more explicit?

**Comment by MakisH** [comment]
I have the same issue, only with Messenger. Some interesting notes:

1. I get sound, I get a red taskbar indicator and number on unread notifications. I only don't see the baloon (only for Messenger).
2. I was using Franz before and I had the same problem, also only with Messenger.
3. I get desktop notifications normally through Messenger on Firefox 50.

Of course I have desktop notification set to "enabled" for Messenger and "Don't Disturb mode" to "off". I tried to remove and re-add the service.

My Rambox about:
Version: 0.4.5
Platform: linux (x64)
Electron: 1.4.7
Chromium: 53.0.2785.143
Node: 6.5.0
I am using Ubuntu 16.04.

The following may not be correct, but I have a feeling that in the beginning it was working and it stopped after I tried to play around with the "do not disturb" mode. I am using it only for 1-2 days now.
